### PR TITLE
Issue 72/user list management, Issue 73/refactoring, Issue 76/application routing

### LINKED ISSUE #72: User list management
**Is your feature request related to a problem? Please describe.**
Will want to create list of users that people can manage (i.e. add or delete from list)

**Describe the solution you'd like**
Create a new Form component that could manage CRUD-like operations for adding/deleting users from a user's list on their Pod

**Describe alternatives you've considered**
None

**Additional context**
None

### LINKED ISSUE #73: Refactoring session-core, session-helper, and statusReducer
**Is your feature request related to a problem? Please describe.**
Current state of session-core, session-helper, and statusReducer doesn't seem to be too WET at the moment on Development. However, I want to see if I can make it more DRY before pushing the existing version of Development to Master.

**Describe the solution you'd like**
Would like to comb over code one more time to see what can be refactored, clarified functions with better variable name, etc., especially since #71 had just recently been merged into Development with the ttl file update functionality.

**Describe alternatives you've considered**
None

**Additional context**
None

### LINKED ISSUE #76: Application Routing
**Is your feature request related to a problem? Please describe.**
At the moment, PASS uses a single page to render everything. We're starting to mess around with a home screen, so adding routing to the Application would be a good idea.

**Describe the solution you'd like**
Have already implemented some routing to PASS using React Router. Can expand on that.

### PR DESCRIPTION
**Pull request by leekahung**
Draft PR for Issue #72 (user list manager) and includes Issue #73 (refactoring session-core/session-helper/reducer, etc.) as part of the commits. Issue #76 (application routing) is also included here in preparation for MUI and multi-page support.

Improvements here includes the addition of a user list manager and login tracker (both of which could be subject to change as is) in preparation for a home screen. Functionality and components build here could be used for that.

The user list can only be accessed by the user themselves within Pod container Users and shows the data for a user's name and their Pod URL.

The login tracker only shows the time and date of their latest login and is placed within public, which only provides read access to other users.

Combined these two functionalities provides a list of users that could be seen from their main page the other user's name, the other user's  Pod URL, and their Last Active Date.

### CODE REVIEW

**Comment by leekahung** [comment]
Updating PR to include Issue #76. Likely be the final changes before merging to Development, then we'll move onto #75 with Material UI as discussed.

File: src/components/Forms.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { Logout } from './Login';
+
+/**
+ * Forms Component - Component that generates Forms section for PASS
+ *
+ * @memberof GlobalComponents
+ * @name Forms
+ */
+
+const Home = () => (
```
Comment (leekahung) [inline src/components/Forms.jsx]: Line 19 needs to be changed to Forms.

File: src/components/Forms.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <UploadDocumentForm />
+    <FetchDocumentForm />
+    <DeleteDocumentForm />
+    <SetAclPermissionForm />
+    <CrossPodQueryForm />
+    <CrossPodWriteForm />
+  </>
+);
+
+export default Home;
```
Comment (leekahung) [inline src/components/Forms.jsx]: Line 31 needs to be changed to Forms.

File: src/contexts/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import SelectUserContext from './selectUserContext';
+import UserListContext from './userListContext';
+
```
Comment (leekahung) [inline src/contexts/index.js:4]: In file, you're missing the `import RouterContext from './routerContext'` for imports and exports

**Review by leekahung — COMMENTED** [review]
Please see the reviews and change the files accordingly.

File: src/components/Forms.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Forms.jsx]: Resolved this

File: src/components/Forms.jsx (hunk 2 above)
Comment (leekahung) [reply src/components/Forms.jsx]: Resolved this

File: src/contexts/index.js (hunk 3 above)
Comment (leekahung) [reply src/contexts/index.js:4]: Got this import corrected to index.js instead of from src/contexts/routerContext.js

**Review by Jared-Krajewski — APPROVED** [review]

**Comment by Jared-Krajewski** [comment]
went over and tested changes in discord voice chat. 🚀

**Pull request merged**